datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -2282,6 +2294,13 @@ select max(x_dict) from value_dict group by x_dict % 2 order by max(x_dict);
 4
 5
 
+statement ok
+drop table value
```

Comment:
Drive By cleanup: added clean up from the test, which I noticed while working on this PR